### ISSUE TITLE
why not softmax

### ISSUE THREAD

**Issue by rcx986635** [PuNeal/DLDL-v2-PyTorch#2]
Firstly thank you for your share code, then I'm confused why  the last layer is  sigmoid + L1-normal ,why not softmax ?  thanks again

**Comment by rcx986635** [comment]
and the figure 2 in paper ,has softmax in the framework..

**Comment by rcx986635** [comment]
@PuNeal

**Comment by yjl450** [comment]
I'm confused about the same problem. Which did you use eventually? @rcx986635

**Comment by Mudwall** [comment]
@rcx986635@yjl450
I get about 2.4(MAE) with this Pytorch implementation. I try Sigmoid and Softmax both. The paper result (1.97) is confusing.
